src/languages/id-ID.js

```diff
@@ -379,8 +379,8 @@ export default {
     smtpDkimheaderFieldNames: "Header Keys untuk ditambahkan (Optional)",
     smtpDkimskipFields: "Header Keys not untuk ditambahkan (Optional)",
     wayToGetPagerDutyKey: "Anda dapat menambahkan melalui Service -> Service Directory -> (Select a service) -> Integrations -> Add integration. Lalu Anda dapat menjadi dengan kata kunci \"Events API V2\". Informasi tambahan {0}",
-    "Integration Key": "Integration Key",
-    "Integration URL": "Integration URL",
+    "Integration Key": "Kunci Integrasi",
+    "Integration URL": "Url Integrasi",
```

Comment:
```suggestion
    "Integration URL": "URL Integrasi",
```
URL should be capitalized probably as it is an acronym